### PR TITLE
added randomizer and splice to prevent duplicate results

### PR DESCRIPTION
**Pull request by connietee22**
- Added the standard Fisher-Yates formula to shuffle entire array
- used splice to keep only the first 10 results in array

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -258,7 +258,20 @@ form button:focus {
     min-height: 100%;
 }
 
+/* loading spinner in the event API map call is delayed */
+.loadingSpinner {
```
Comment (kdlry) [inline src/App.css:262]: Nice touch!

**Review by kdlry — APPROVED** [review]
Looks good - approved by me :)

File: src/App.js [hunk 2]
```diff
@@ -115,19 +112,33 @@ class App extends Component {
...
-        // randomize all numbers
-        // use for counter to just push 10 shops
-        for (let i = 0; i < 10; i++) {
-          const randomCoffeeShopIndex = Math.floor(Math.random() * returnedCoffeeShops.length);
-          randomCoffeeShops.push(returnedCoffeeShops[randomCoffeeShopIndex]);
+        
+        // creating a copy of the array to randomize and reduce to 10
+        let randomCoffeeShops = [...returnedCoffeeShops]
+
+        // standard fisher-yates randomizer to randomize entire array and prevent duplicates
```
Comment (afacey) [inline src/App.js:119]: Nice randomization!

**Review by afacey — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (raphaelroques) [reply src/App.css:262]: approved

**Pull request merged**